src/index.css

```diff
@@ -2,4 +2,251 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  background-color: #fafafa;
+}
+
+:root {
+  --primary-color: #0290ff;
+  --primary-dark-color: #121212;
+  --gray-color: #e8e8e8;
+  --primary-font: 'Montserrat', serif;
+  --secondary-font: 'RobotoSlab', sans-serif;
+}
+
+header {
+  height: 2rem;
+  margin: 0 0 2.375rem;
+  padding: 1.4rem 2rem 1.6rem 2rem;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.log {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: var(--primary-font);
+  font-size: 1rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.menu-hamberger > div {
+  height: 3px;
+  width: 2rem;
+  background: #0290ff;
+  margin: 5px 0;
+  border-radius: 8px;
+}
+
+.profile a span {
+  display: none;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+form input {
+  height: 1.6rem;
+  margin: 1rem 1rem 0 0;
+  padding: 0.813rem 0.813rem 0.813rem 1.063rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  font-family: var(--primary-font);
+  font-size: 1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #c4c4c4;
+}
+
+form button {
+  margin: 1.188rem 0 0 0;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  text-transform: uppercase;
+  width: 11.5rem;
+  font-family: var(--secondary-font);
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.new-book {
+  margin: 0 0 2.375rem;
+  padding: 1rem 1rem 1rem 1rem;
+}
+
+.new-book h2 {
+  height: 1.5rem;
+  margin: 1.813rem 0.75rem 1.188rem 0;
+  font-family: var(--primary-font);
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+@media screen and (min-width: 768px) {
+  header {
```

Comment:
- [ ] Your layouts are overlapped. It would be nicer to fix that to improve you UI design. Please notice that my screen size is 1020px.

<img width="850" alt="image" src="https://user-images.githubusercontent.com/77190203/193407264-f6f9f220-2b51-4521-a2c0-9f287ba30605.png">